torch/csrc/api/include/torch/nn/functional/normalization.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <torch/nn/options/normalization.h>
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor normalize(
+    const Tensor& input,
+    const NormalizeOptions& options,
```

Comment:
When we confirm that `options = {}` doesn't cause the problem, I believe we should change it back to `options = {}`, and it would be awesome to have a test for the `F::normalize(input)` (i.e. no options) use case as well :D Thanks so much for your help @gtamba !

Reply:
@yf225  : Can I still make these changes? I notice that facebook-bot is trying to land the pull request, and I didn't add the test for no options yet

Reply:
@gtamba : Yes sorry this is my fault - didn't notice this before trying to land yesterday :( I saw you updated the PR (thanks a lot :D) and I will try to land it again